### PR TITLE
Feature/leaf1495 routing system place in absolute paths

### PR DESCRIPTION
**Pull request by jeremyrl**

### CODE REVIEW

File: LEAF_Nexus/admin/templates/site_elements/generic_plainDialogLarge.tpl [hunk 1]
```diff
@@ -15,5 +15,7 @@ $(function() {
 		width: $(window).width() * 0.8,
 		height: $(window).height() * 0.8
 	});
+    $('#plainDialogLarge').dialog({minWidth: ($(window).width() * .8) + 30});
```
Comment (thomasnguyen95) [inline LEAF_Nexus/admin/templates/site_elements/generic_plainDialogLarge.tpl:18]: Did routing require changing the style of a dialog?

File: LEAF_Nexus/js/dialogController.js [hunk 2]
```diff
@@ -18,10 +18,15 @@ function dialogController(containerID, contentID, indicatorID, btnSaveID, btnCan
 	this.validatorOks = new Object();
 	this.invalid = 0;
 
+	//calculate min width of dialog based on min width of content div
+	var minWidth = parseInt($('#' + this.contentID).css('min-width'));
```
Comment (thomasnguyen95) [inline LEAF_Nexus/js/dialogController.js:22]: Did routing require changing the style of a dialog?

File: LEAF_Request_Portal/admin/templates/mod_groups.tpl [hunk 3]
```diff
@@ -483,7 +512,13 @@ $(function() {
 	dialog_simple = new dialogController('simplexhrDialog', 'simplexhr', 'simpleloadIndicator', 'simplebutton_save', 'simplebutton_cancelchange');
 
 	$('#simpleloadIndicator').css({width: $(window).width() * .78, height: $(window).height() * .78});
+
+    dialog_simple.setCancelHandler(function(){
+        $('#simplexhr').css({width: $(window).width() * .8, height: $(window).height() * .8});
+        dialog_simple.setTitle("");
+    });
 	$('#simplexhr').css({width: $(window).width() * .8, height: $(window).height() * .8});
+    $('#simplexhrDialog').dialog({minWidth: ($(window).width() * .78) + 30});
```
Comment (dogwang94) [inline LEAF_Request_Portal/admin/templates/mod_groups.tpl:521]: indentation?

File: LEAF_Nexus/admin/templates/site_elements/generic_plainDialogLarge.tpl (hunk 1 above)
Comment (jeremyrl) [reply LEAF_Nexus/admin/templates/site_elements/generic_plainDialogLarge.tpl:18]: This has dev merged into it. So there are some changes not related to routing.

File: LEAF_Nexus/js/dialogController.js (hunk 2 above)
Comment (jeremyrl) [reply LEAF_Nexus/js/dialogController.js:22]: This has dev merged into it. So there are some changes not related to routing.

File: LEAF_Request_Portal/admin/templates/mod_groups.tpl (hunk 3 above)
Comment (jeremyrl) [reply LEAF_Request_Portal/admin/templates/mod_groups.tpl:521]: This has dev merged into it. So there are some changes not related to routing. This is the indentation present in dev.

**Pull request merged**